webapp/src/App.tsx

```diff
@@ -127,6 +128,14 @@ export default class App extends React.Component<Props> {
                                 <UtteranceDetail />
                               </BasicLayout>
                             </Route>
+                            <Route
+                              path="/:jobId/performance_analysis_comparison"
+                              exact
+                            >
+                              <BasicLayout>
+                                <PerformanceAnalysisComparison />
+                              </BasicLayout>
+                            </Route>
```

Comment:
So this page is not necessarily about comparison, right? It works with a single model too, and might be useful to get more vertical space and see more rows. Therefore, I would remove the word "Comparison" everywhere, including file names.
```suggestion
                            <Route
                              path="/:jobId/performance_analysis"
                              exact
                            >
                              <BasicLayout>
                                <PerformanceAnalysis />
                              </BasicLayout>
                            </Route>
```